apps/server/src/utils/ElasticSearchClient.ts

```diff
@@ -0,0 +1,100 @@
+import type { estypes } from '@elastic/elasticsearch'
+import { Client } from '@elastic/elasticsearch'
+import Mock from '@elastic/elasticsearch-mock'
+import { readFileSync } from 'fs'
+import { resolve } from 'path'
+import config from '../config'
+
+import type { CachedInstitution } from 'src/shared/contract'
+
+function getInstitutionFilePath () {
+  if (config.Env === 'test') {
+    console.log('loading test institutions')
```

Comment:
If we feel like we need to log, then we should use the logger, so we can disable them for our tests. Whenever we run tests right now they are really polluted with log statements, and it makes it difficult to fix things that actually matter.